integrations/_infra/Dockerfile

```diff
@@ -1,28 +1,58 @@
-FROM python:3.11-slim-bookworm
+FROM python:3.11-alpine AS base
 
 ARG BUILD_CONTEXT
+
+ENV LIBRDKAFKA_VERSION=1.9.2
+
+# Install system dependencies and libraries
+RUN apk add --no-cache \
+    gcc \
+    musl-dev \
+    build-base \
+    bash \
+    oniguruma-dev \
+    make \
+    autoconf \
+    automake \
+    libtool \
+    curl \
+    libffi-dev && \
+        echo "@edge http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories \
+        && echo "@edgecommunity http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories \
+        && apk add --no-cache alpine-sdk "librdkafka@edgecommunity>=${LIBRDKAFKA_VERSION}" "librdkafka-dev@edgecommunity>=${LIBRDKAFKA_VERSION}" && curl -sSL https://install.python-poetry.org | python3 - \
```

Comment:
why not using `librdkafka-dev`

Reply:
Wouldn't compile at the poetry install phase.

Reply:
worked for the agent, what is the difference? if you could share the error?

Reply:
```
4.469   In file included from /tmp/tmp425ca7xe/confluent-kafka-2.5.3/src/confluent_kafka/src/Admin.c:17:
4.469   /tmp/tmp425ca7xe/confluent-kafka-2.5.3/src/confluent_kafka/src/confluent_kafka.h:66:2: error: #error "confluent-kafka-python requires librdkafka v2.5.3 or later. Install the latest version of librdkafka from the Confluent repositories, see http://docs.confluent.io/current/installation.html"
4.469      66 | #error "confluent-kafka-python requires librdkafka v2.5.3 or later. Install the latest version of librdkafka from the Confluent repositories, see http://docs.confluent.io/current/installation.html"
```